make/docker-compose.clair.tpl

```diff
@@ -41,7 +41,7 @@ services:
     depends_on:
       - postgres
     volumes:
-      - ./common/config/clair:/config:z
+      - ./common/config/clair/config.yaml:/etc/clair/config.yaml:z
```

Comment:
Just want to make sure there is no other files under ./common/config/clair  since we only map the config.yaml